unitary/alpha/sparse_vector_simulator.py

```diff
@@ -0,0 +1,121 @@
+"""Noiseless simulator using a sparse state vector.
+
+Just enough features to support Unitary are implemented.
```

Comment:
I think I would clarify this statement and say what features are supported.  It looks like this supports all cirq gates that have unitary actions as well as the post selection operator.  Correct?

Reply:
Correct.